src/cf-forms/src/molecules/form-fields.less

```diff
@@ -11,15 +12,19 @@
     &__checkbox,
     &__radio {
         .a-label {
+            display: table-row;
+            display: flex;
             vertical-align: middle;
             cursor: pointer;
 
             &:before {
-                display: inline-block;
+                display: table-cell;
+                align-self: flex-start;
 
                 border: 1px solid @input-border;
-                height: unit( 18px / @base-font-size-px, em );
-                width: unit( 18px / @base-font-size-px, em );
+                min-width: unit( 20px / @base-font-size-px, em);
```

Comment:
Hmm, for some reason I needed this to prevent flexbox from stretching the size of the checkbox/radio icon, but seems to work now without it.